lib/piezo.js

```diff
@@ -401,16 +401,37 @@ Piezo.ToSong = function(stringSong, beats) {
   return song;
 };
 
+/**
+ * Play a note for a duration.
+ * @param {string} note - see Piezo.Notes.  Case-insensitive.
+ *   If an octave is not given, the configured default octave
+ *   will be used.
+ * @param {number} duration - in milliseconds.
+ */
 Piezo.prototype.note = function(note, duration) {
-  var tone = Piezo.Parsers.hzFromInput(note);
-
-  return this.tone(tone, duration);
+  var frequency = Piezo.Parsers.hzFromInput(note);
+  return this.frequency(frequency, duration);
```

Comment:
Stylistically, this could all be done in one statement, e.g.:

```
return this.frequency(Piezo.Parsers.hzFromInput(note), duration);
```

as you do in `frequency`, eliminating the need for the `var` declaration and being consistent, but that's totally up to you.